apps/condo/domains/organization/components/CreateEmployeeForm.tsx

```diff
@@ -96,6 +100,18 @@ export const CreateEmployeeForm: React.FC = () => {
         role: get(employeeRoles, [0, 'id'], ''),
     }
 
+    const [selectedRoleId, setSelectedRoleId] = useState(initialValues.role)
+    const selectedRole = useMemo(() => employeeRoles.find(x=> x.id === selectedRoleId), [selectedRoleId])
```

Comment:
add readable virable names

Reply:
@AntonAL ```find(x=> x.id === selectedRoleId)```

Reply:
@MrFoxPro, please, use meaningful variable names.

Reply:
Refactored in `dfba0dd80`